test/uk/gov/hmrc/incometaxpenaltiesfrontend/views/IndexViewSpec.scala

```diff
@@ -70,7 +72,8 @@ class IndexViewSpec extends AnyWordSpec with Matchers with GuiceOneAppPerSuite w
                 lspCardData = Seq(),
                 lppCardData = Seq(),
                 penaltiesOverviewViewModel = PenaltiesOverviewViewModel(Seq(), hasFinancialCharge = false),
-                isAgent = isAgent
+                isAgent = isAgent,
+                actionsToRemoveLinkDate = Some(LocalDate.of(2028,4,1))
```

Comment:
set date as val as it's being used in multiple places

`private val somePocDate: Option[LocalDate] = Some(LocalDate.of(2028, 4, 1)) `

```suggestion
                actionsToRemoveLinkDate = somePocDate
```